src/ol/layers/MapboxLayer.js

```diff
@@ -249,6 +249,9 @@ export default class MapboxLayer extends Layer {
          * @type {string}
          */
         this.copyright = getCopyrightFromSources(this.mbMap);
+        this.dispatchEvent({
+          type: 'change:copyright',
+        });
```

Comment:
Now I remember why putting copyright in the properties object was a good idea:
https://github.com/geops/mobility-toolbox-js/pull/14/files

Revert this change in mobility-toolbox

Reply:
using set/get method for object properties doesn't seems to be the correct way.
Then copyright property are not corectly set to the layer anymore